src/qasm/parser/parser.c

```
@@ -0,0 +1,256 @@
+    void** b;
+
+    // Determines the instruction type based on the string hash.
+    switch(instructionHash) {
+        case 2985:
+            instruction->opCode = BLOCK_SWAP;
+            b = malloc(sizeof(void*));
+            parseInt32(b, 0, buff[1]);
+
+            instruction->params = b;
+            instruction->paramCount = 1;
+            break;
```

cppcheck (2.10-2)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)